plugins/modelposepublisher/include/gazebo/ModelPosePublisher.hh

```diff
@@ -107,7 +107,7 @@ namespace gazebo
     ///
     /// This plugin __does not__ support <a href="http://gazebosim.org/tutorials?tut=nested_model&cat=build_robot">Gazebo nested models</a>
     /// since the method `gazebo::physics::WorldPose()` does not work properly
-    /// with nested models.
```

Comment:
This works fine, but consider that Doxygen also support Markdown, so you can also write markdown-style links.